### PR TITLE
1.0.0-b37

### PR DESCRIPTION
**Pull request by vinniefalco**
* CMake hide command lines in .vcxproj Output windows"
* Rename to detail::is_invocable
* Rename project to http-bench

WebSocket:

* Add is_upgrade() free function
* Document websocket::stream thread safety
* Rename to websocket::detail::pausation

API Changes:

* Provide websocket::stream accept() overloads
* Refactor websocket decorators
* Move everything in basic_fields.hpp to fields.hpp
* Rename to http::dynamic_body, consolidate header

### CODE REVIEW

File: include/beast/core/detail/is_call_possible.hpp [hunk 1]
```diff
@@ -13,41 +13,6 @@
 namespace beast {
```
Comment (nbougalis) [inline include/beast/core/detail/is_call_possible.hpp]: This file now appears empty, save for the `#include <type_traits>`. Given that it's a `detail`, should it just be entirely removed?

File: include/beast/websocket/detail/pausation.hpp [hunk 2]
```diff
@@ -19,9 +19,12 @@ namespace beast {
 namespace websocket {
 namespace detail {
 
-// "Parks" a composed operation, to invoke later
+// A container to hold a suspended, asynchronous composed
+// operation that has been suspended. The contained object
```
Comment (nbougalis) [inline include/beast/websocket/detail/pausation.hpp]: "A container to hold a suspended, ... that has been suspended".

File: include/beast/core/detail/is_call_possible.hpp (hunk 1 above)
Comment (vinniefalco) [inline include/beast/core/detail/is_call_possible.hpp]: Whoops... yes. Its probably not even included anywhere anymore. Or at least, it shouldn't be.
willfix

File: include/beast/websocket/detail/pausation.hpp (hunk 2 above)
Comment (vinniefalco) [inline include/beast/websocket/detail/pausation.hpp]: Good thing its not a public interface... lol
willfix

**Pull request merged**